src/main/java/org/folio/rest/impl/InvoiceLinesHelper.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.rest.impl;
+
+import static org.folio.invoices.utils.HelperUtils.handleGetRequest;
+import static org.folio.invoices.utils.ResourcePathResolver.resourceByIdPath;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+
+import org.folio.rest.jaxrs.model.InvoiceLine;
+
+import io.vertx.core.Context;
+
+public class InvoiceLinesHelper extends AbstractHelper {
+
+  InvoiceLinesHelper(Map<String, String> okapiHeaders, Context ctx, String lang) {
+    super(getHttpClient(okapiHeaders), okapiHeaders, ctx, lang);
+  }
+
+  public CompletableFuture<InvoiceLine> getInvoiceLines(String id) {
+    CompletableFuture<InvoiceLine> future = new VertxCompletableFuture<>(ctx);
+    try {
+      String endpoint = String.format(resourceByIdPath("invoiceLines") + id, lang);
```

Comment:
1. Please use `ResourcePathResolver.INVOICE_LINES`
2. As far as I can see the result of `resourceByIdPath("invoiceLines") + id` is string like `"/invoice-storage/invoice-lines/some-id"`. This means `String.format` will not be able to do anything